service/src/main/java/bio/terra/cbas/dao/RunSetDao.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cbas.dao;
+
+import bio.terra.cbas.models.RunSet;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class RunSetDao {
+
+  private final NamedParameterJdbcTemplate jdbcTemplate;
+
+  @Autowired
+  public RunSetDao(NamedParameterJdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  public int createRunSet(RunSet runSet) {
+    return jdbcTemplate.update(
+        "insert into run_set (id, method_id) values (:id, :methodId)",
+        new BeanPropertySqlParameterSource(runSet));
+  }
+
+  public Optional<RunSet> getRunSet(UUID runSetId) {
```

Comment:
Since nothing is using this yet, I'll remove. Presumably whoever comes to re-implement this will have more context for deciding whether to make it optional or not... and maybe even have more framework around it to write a sensible test